spec/rubocop/cop/rspec/factory_bot/excessive_create_list_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Rubocop::Cop::RSpec::FactoryBot::ExcessiveCreateList do
+  let(:cop_config) do
+    { 'MaxAmount' => max_amount }
+  end
+
+  context 'when MaxAmount is set to 50' do
+    let(:max_amount) { 50 }
```

Comment:
Is there a reason not to use the default 10 here?

Reply:
None whatsoever 😆. I'll set it to the default.